BusinessLayer/DataPopulators/JsonDefines.h

```diff
@@ -112,14 +112,14 @@ namespace JsonFormat
     const QString BATTERYFAULTS_CELLUNDERVOLTAGE = "CellUnderVoltage";
     const QString BATTERYFAULTS_CELLOVERTEMP = "CellOverTemp";
     const QString BATTERYFAULTS_MEASUREMENTUNTRUSTED = "MeasurementUntrusted";
-    const QString BATTERYFAULTS_CMUCOMMTIMEOUT = "CMUCommTimeout";
+    const QString BATTERYFAULTS_CmuCOMMTIMEOUT = "CmuCommTimeout";
```

Comment:
The Json defines still have to be in all Caps, since the key in the Json packet comes in as `CMUCommTimeout`